### PR TITLE
Update danish translation (v.1.0.7)

### PR DESCRIPTION
**Pull request by Stisen1**
- Focused translations on hero- and town screen windows

### CODE REVIEW

File: files/lang/dk.po [hunk 1]
```diff
@@ -8019,19 +8046,19 @@ msgid "The three Anduran artifacts magically combine into one."
 msgstr ""
 
 msgid "View Spells"
-msgstr ""
+msgstr "Vis besværgelser"
 
 msgid "View %{name} Info"
-msgstr ""
+msgstr "Vis %{name} Info"
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:8052]: Yes.

File: files/lang/dk.po [hunk 2]
```diff
@@ -6357,32 +6372,34 @@ msgid "Select town to port to."
 msgstr ""
 
 msgid "Enemy heroes are now fully identifiable."
-msgstr ""
+msgstr "Fjendtlige helte er nu fuldt identificerbar."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:6375]: Yes! You're right! It has been corrected

File: files/lang/dk.po [hunk 3]
```diff
@@ -2559,16 +2558,16 @@ msgid "Are you sure you want to dismiss this army?"
...
-msgstr ""
+msgstr "Opgrader dine tropper."
 
 msgid "Dismiss"
-msgstr ""
+msgstr "Bortvis"
 
 msgid "Dismiss this army."
-msgstr ""
+msgstr "Bortvis din hær."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:2570]: Had a screaming baby at my ear, when I did this. It has been corrected.

File: files/lang/dk.po [hunk 4]
```diff
@@ -2559,16 +2558,16 @@ msgid "Are you sure you want to dismiss this army?"
 msgstr "Er du sikker på du vil bortvise denne hær?"
 
 msgid "Upgrade"
-msgstr ""
+msgstr "Opgrader"
 
 msgid "Upgrade your troops."
-msgstr ""
+msgstr "Opgrader dine tropper."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:2564]: same correction as above mentionen

File: files/lang/dk.po [hunk 5]
```diff
@@ -2559,16 +2558,16 @@ msgid "Are you sure you want to dismiss this army?"
 msgstr "Er du sikker på du vil bortvise denne hær?"
 
 msgid "Upgrade"
-msgstr ""
+msgstr "Opgrader"
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:2561]: I thought of this as I translated it. I wasn't certain of which context this word would appear in inside the game... It has been corrected to the imperative tense.

File: files/lang/dk.po [hunk 6]
```diff
@@ -717,7 +717,6 @@ msgstr "Viser resultater for individuelle monstres handlinger."
 msgid "Are you sure you want to finish the battle in auto mode?"
 msgstr "Er du sikker på du vil færdiggøre denne kamp automatisk?"
 
-#, fuzzy
 msgid "Are you sure you want to enable auto combat?"
 msgstr "Er du sikker på, at du vil flygte?"
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:721]: Wierd how this happens... It has been corrected.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Stisen1 , thank you very much for this change!

**Pull request merged**